version_control/katalyst_by_codurance/css/modules/services-cards.css

```diff
@@ -1,117 +1,111 @@
 {% import '../tools/_breakpoints.css' as breakpoints %}
-
-.services-cards {
-    margin-top: var(--space-l);
-}
+{% import '../tools/_macros.css' as macros %}
 
 .service-cards__card {
-    padding: var(--space-l);
+    padding: var(--space-xl) var(--space-l);
     background-color: white;
-    border-radius: 20px;
+    {{ macros.card_styles() }}
+    position: relative;
+    overflow: hidden;
 }
 
-{% call breakpoints.small_and_medium() %}
-.services-cards article:not(:first-child) {
-    margin-top: var(--space-m);
+.service-cards__card img {
+    position: absolute;
 }
-{% endcall %}
-
-{% call breakpoints.large_and_extra_large() %}
-    .services-cards {
-        display: grid;
-        grid-template-columns: repeat(12, 1fr);
-        gap: var(--space-m);
-    }
-
-    .col-3 {
-        grid-column: span 3;
-    }
 
-    .col-4 {
-        grid-column: span 4;
-    }
-
-    .service-cards__card {
-        align-self: start;
-        transition: scale .2s,
-                    outline .2s;
-    }
-{% endcall %}    
-
-
-
-.service-cards__card:hover,
-.service-cards__card:focus {
-    outline: 2px solid var(--orange);
-    scale: 1.05;
-    transition: scale .3s,
-                outline .2s; 
+/* Positioning of card inner graphics */
+.service-cards__card:nth-child(1) img {
+    left: -6%;
+    top: -32%;
 }
```

Comment:
I haven't tried but having a custom css variables for every position it might help to manage easier every position of the bg triangle on each card. Also, you can make sure that if something happend you have a default position